mousestyles/path_diversity/clean_movements.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
+
+def clean_movements(movements, keep_index=None):
+    r"""
+    Returns a list of cleaned movement objects.
+
+    For each element of the input `movements` object, detects the existence of
+    rows which have i) same x,y-coordinates, or ii) same timestamps.
+    For case i), and removes the duplicated rows except for the first.
+    For case ii), raises value error.
+
+    Parameters
+    ----------
+    movements : list
+        each element is pandas.DataFrame containing 
+        CT, CX, CY coordinates.
+        Should have length greater than 1.
+
+    kee_index : boolean
+        whether or not keep the original index.
+        Deafalt is False.
+        If false, cleaned movement object is re-indexed.
+
+    Returns
+    -------
+    cleaned-movements : list
+        each element is cleaned movement object.
+
+    Examples
+    --------
+    >>> strain_nums = [0,0,1,1,2,2]
+    >>> mouse_nums = [0,0,1,1,2,2]
+    >>> day_nums = [0,11,0,11,0,11]
+    >>> movements = [data.load_movement(strain_num, mouse_num, day_num) for \
+    >>> strain_num, mouse_num, day_num in zip(strain_nums, mouse_nums, day_nums)]
+    >>> cleaned_movements, num_same_locs = cleaning_movements(movements)
```

Comment:
We want the function to be `clean_movements(movements)` ... not "cleaning".
Also, I'm not sure what the purpose of `num_same_locs` is. Could you explain?